smac/facade/smac_ac_facade.py

```diff
@@ -361,12 +370,27 @@ def __init__(
                 **acq_def_kwargs,
             )
 
+        if user_priors:
+            # a solid default value for decay_beta - empirically founded
+            default_beta = scenario.ta_run_limit / 10
+            decay_beta = acq_def_kwargs.get("decay_beta", default_beta)
+            discretize = isinstance(model_instance, RandomForestWithInstances) or isinstance(
+                model_instance, RFRImputator
+            )
+            acquisition_function_instance = PriorAcquisitionFunction(
+                acquisition_function=acquisition_function_instance, **user_prior_kwargs, **acq_def_kwargs
```

Comment:
Both decay_beta and discretize are not used. Should they be part of "user_prior_kwargs"?

Reply:
After this issue is fixed, we can merge it. :)

Reply:
Great to hear! I'll check it right away.

Reply:
Addressed the issue now, thanks for spotting.